src/components/profile/ProfileTeamLocation.vue

```diff
@@ -286,6 +284,10 @@ export default {
     display: flex;
     margin: 2em 0;
   }
+  .profile__team-location > * {
```

Comment:
Can we come up with a better way to style the immediate contents of ```profile__team-location```? The ```*``` operator is not a great selector because it has no specificity and it's very fragile for future code. It's also not very descriptive.